frontend/src/bundles/talent-onboarding/components/contacts-cv-step/contacts-cv-step.tsx

```diff
@@ -0,0 +1,348 @@
+import {
+    FileUpload,
+    FormControl,
+    FormHelperText,
+    FormLabel,
+    Grid,
+    Input,
+    Typography,
+} from '~/bundles/common/components/components.js';
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+import { useAppForm } from '~/bundles/common/hooks/hooks.js';
+
+// import { useAppDispatch, useAppForm } from '~/bundles/common/hooks/hooks.js';
+// import { type RootReducer } from '~/framework/store/store.package.js';
+// import { useFormSubmit } from '../../context/form-submit-provider.context.js';
+import { validateFileSize } from '../../helpers/validate-file-size.js';
+// import { actions } from '../../store/talent-onboarding.js';
+import { type ContactsCVStepDto } from '../../types/types.js';
+import { contactsCVStepValidationSchema } from '../../validation-schemas/validation-schemas.js';
+import {
+    ACCEPTED_CV_TYPES,
+    ACCEPTED_PHOTO_TYPES,
+    DEFAULT_CONTACTS_CV_STEP_PAYLOAD,
+    REQUIRED,
+} from './constants/constants.js';
+import styles from './styles.module.scss';
+
+const ContactsCVStep: React.FC = () => {
+    //delete this after the comments have been uncommented.
+    const { control, errors, watch, setError } = useAppForm({
+        defaultValues: DEFAULT_CONTACTS_CV_STEP_PAYLOAD,
+        validationSchema: contactsCVStepValidationSchema,
+    });
+
+    // const savedPayload = useSelector(
+    //     (state: RootReducer) => state.talentOnBoarding.contactsCVStep,
+    // );
+
+    // const { control, handleSubmit, errors, setError, watch } =
+    //     useAppForm<ContactsCVStepDto>({
+    //         defaultValues: { ...savedPayload },
+    //         validationSchema: contactsCVStepValidationSchema,
+    //     });
+
+    // const { setSubmitForm } = useFormSubmit();
+
+    // const dispatch = useAppDispatch();
+
+    // const onSubmit = useCallback(
+    //     async (data: ContactsCVStepDto): Promise<boolean> => {
+    //         await dispatch(actions.contactsCVStep(data));
+    //         return true;
+    //     },
+    //     [dispatch],
+    // );
+
+    // useEffect(() => {
+    //     setSubmitForm(() => {
+    //         return async () => {
+    //             let result = false;
+    //             await handleSubmit(async (formData) => {
+    //                 result = await onSubmit(formData);
+    //             })();
+    //             return result;
+    //         };
+    //     });
+    //     return () => {
+    //         setSubmitForm(null);
+    //     };
+    // }, [handleSubmit, onSubmit, setSubmitForm]);
+
+    const [photoURL, setPhotoURL] = useState<string>('');
+
+    const handlePhotoFileChange = useCallback(
+        (field: ControllerRenderProps<ContactsCVStepDto, 'photo'>) =>
+            (event: React.ChangeEvent<HTMLInputElement>): boolean => {
+                if (!event.target.files) {
+                    return false;
+                }
+
+                const [file] = [...event.target.files];
```

Comment:
I think we can do just:
```suggestion
                const [file] = event.target.files;
```